### PR TITLE
Add Issue and PR templates

### PR DESCRIPTION
**Pull request by philberty**
This removes the greetings automation and adds in PR and issue templates in github. These were taken from rust-clippy as a base and changed for GCC Rust.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ice.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+name: Internal Compiler Error
+about: Create a report for an internal compiler error in Clippy.
```
Comment (flip1995) [inline .github/ISSUE_TEMPLATE/ice.md:3]:
```suggestion
about: Create a report for an internal compiler error in gccrs.
```

**Review by flip1995 — APPROVED** [review]
One paperclip slipped through.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+automatically closed when your PR is merged. If your change addresses
+an issue but does not fully fix it please mark it as "Addresses #issue_number"
+in the git commit message.
+
+Here is a checklist to help you with your PR.
+
+- \[ ] Read contributing guidlines
+- \[ ] `make check-rust` passes locally
+- \[ ] Run `clang-format`
+- \[ ] Added any relevant test cases to gcc/testsuite/rust.test
```
Comment (bjorn3) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Can you add an entry for copyright assignment?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+If your PR fixes an issue, you can add "Fixes #issue_number" into this
+PR description and the git commit message. This way the issue will be
+automatically closed when your PR is merged. If your change addresses
+an issue but does not fully fix it please mark it as "Addresses #issue_number"
+in the git commit message.
+
+Here is a checklist to help you with your PR.
+
+- \[ ] Read contributing guidlines
```
Comment (bjorn3) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Why the backslashes?

Comment (philberty) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I am not sure but its copied from https://raw.githubusercontent.com/rust-lang/rust-clippy/master/.github/PULL_REQUEST_TEMPLATE.md

File: .github/ISSUE_TEMPLATE/ice.md (hunk 1 above)
Comment (philberty) [reply .github/ISSUE_TEMPLATE/ice.md:3]: good find

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (philberty) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: good point will do

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (flip1995) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: https://github.com/rust-lang/rust-clippy/pull/6175

Seems that remark complained about those. Not sure why, but this was the easiest way to fix our CI.

File: .github/workflows/remark.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      uses: actions/setup-node@v1.4.4
+
+    - name: Install remark
+      run: npm install remark-cli remark-lint remark-lint-maximum-line-length remark-preset-lint-recommended
+
+    # Run
+    - name: Check *.md files
+      run: git ls-files -z '*.md' | xargs -0 -n 1 -I {} ./node_modules/.bin/remark {} -u lint -f > /dev/null
+
+  # These jobs doesn't actually test anything, but they're only used to tell
```
Comment (flip1995) [inline .github/workflows/remark.yml:31]: Don't think this is necessary for the Rust GCC bors bot.

File: .github/workflows/remark.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
+name: Remark
+
+on:
+  push:
+    branches:
+      - auto
+      - try
+  pull_request:
+    paths:
+    - '**.md'
```
Comment (flip1995) [inline .github/workflows/remark.yml:10]: You also may want to exclude `gcc/config/**/*.md` so that all the machine description files won't get checked.

**Pull request merged**